src/pages/guides/app_builder_guides/storage/database.md

```diff
@@ -0,0 +1,621 @@
+---
+keywords:
+  - Adobe I/O
+  - App Builder
+  - Database Storage
+  - aio-lib-db
+  - DocumentDB
+  - MongoDB
+title: Database Storage Getting Started
+---
+
+# Overview
+
+Database Storage for App Builder provides document-style database persistence for AIO Runtime Actions. The **aio-lib-db**, which is closely modeled on the MongoDB Database Driver for NodeJS, provides the primary programming interface, while the DB Plugin in the AIO CLI provides additional access.
+
+There is a strict one-to-one relationship between an AIO Project Workspace and a Workspace Database, and each Workspace Database is entirely isolated from all other Workspace Databases.
+
+Before use, a Workspace Database must be explicitly provisioned using either **aio-lib-db** or the AIO CLI. This is a self-service operation subject to organizational quotas and limits.
+
+Workspace Databases are provisioned in one and only one of the following regions: `amer`, `apac`, and `emea`. These acronyms are defined as follows:
+
+- `amer`: North, Central, and South America. Data is stored in the US.
+- `apac`: Asia and Pacific. Data is stored in Japan.
+- `emea`: Europe, the Middle East, and Africa. Data is stored in the EU.
+
+## DB plugin in the AIO CLI
+
+The DB Plugin in the AIO CLI is a utility that facilitates provisioning, initializing, querying, and monitoring Workspace Databases.
+
+The following is only a brief introduction to the DB Plugin. For more thorough documentation see [aio-cli-plugin-app-storage](https://github.com/adobe/aio-cli-plugin-app-storage/tree/epic/abdb-implementation).
+
+### Installation
+
+To be included in aio --version >= 11.0.0
+
+### Region selection
+
+When using the DB Plugin in the AIO CLI, it is important that the region is the same as where the database is provisioned. If not, the connection will fail.
+
+The default region is `amer`, and to set a different one, you can use the `--region` flag or add the `AIO_DB_REGION` variable to your .env file. Supported regions include `amer`, `emea`, and `apac`.
+
+### Provisioning a workspace database
+
+To provision a Workspace Database in the current AIO Project Workspace is as simple as:
+
+```bash
+aio app db provision
+```
+
+The provisioning status can be retrieved with this:
+
+```bash
+aio app db status
+```
+
+To check connectivity with the database:
+
+```bash
+aio app db ping
+```
+
+Additional database commands include:
+
+```bash
+aio app db stats               # Get statistics about your App Builder database
+```
+
+```bash
+aio app db delete              # Delete the database for your App Builder application (non-production only)
```

Comment:
maybe make the descriptions their own line or take descriptions out of commands, so that they are more easily copied